src/main/java/net/sf/mzmine/modules/peaklistmethods/peakpicking/deconvolution/DeconvolutionTask.java

```diff
@@ -59,6 +61,11 @@
     // User parameters
     private final ParameterSet parameters;
 
+	private RSessionWrapper rSession;

```

Comment:
We have several deconvolution methods, but so far only one of them (xcms-based) depends on R.
So I think it makes more sense to move R-related variables into the xcms deconvolution class, instead of this (general) DeconvolutionTask. If we keep it here, there is a slight risk that a problem in R will bring down the whole deconvolution module.